Gems/AtomLyIntegration/EditorModeFeedback/Assets/Shaders/EditorModeCommon.azsli

```diff
@@ -77,3 +83,60 @@ partial ShaderResourceGroup PassSrg : SRG_PerPass_WithFallback
         return CalculateOutputColor(inColor, finalEffect, PassSrg::CalculateFinalBlendAmount(t));
     }
 }
+
+//! Calculates the view position and normal from the fragment position.
+void ComputeViewNormalAndPosition(VSOutput IN, out float3 outPositionWS, out float3 outNormalWS)
```

Comment:
Who is using this function in Editormodefeedback shaders?

Reply:
None of the existing effects, but future effects are free to do so. Particularly, this blog post references this function:

https://github.com/o3de/o3de.org/pull/2380